Volume3

iwiebe7 opened the pull request.

iwiebe7 committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

Test - previous commits didn't go through

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

updates

iwiebe7 committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

updated tags from jan 31 to feb 12

selmaelba committed:
Update volume3.xml

Tagged Theo and Nettie

selmaelba committed:
Update volume3.xml

Edited names

isabelleschlegel committed:
Update volume3.xml

updates from entries 2/10 through 2/16

selmaelba committed:
Update volume3.xml

Divided volume into 4 sections

selmaelba committed:
Merge branch 'volume3' into volume3

selmaelba committed:
Merge pull request #30 from selmaelba/volume3

Volume3

selmaelba committed:
Create Volume16.xml

Tagged parts of first third

selmaelba committed:
Update volume3.xml

Formatted tags for in first fourth

selmaelba committed:
Merge pull request #31 from selmaelba/volume3

Volume3

selmaelba committed:
Update volume3.xml

Formatted tags in first fourth

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

selmaelba committed:
Merge branch 'volume3' into volume3

selmaelba committed:
Merge pull request #32 from selmaelba/volume3

Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

tagging on 2/4

selmaelba committed:
Update volume3.xml

Tagged parts of first fourth

iwiebe7 committed:
Update volume3.xml

selmaelba committed:
Update volume3.xml

Tagged Nov 29

selmaelba committed:
Update volume3.xml

Tagged Nov 26

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

selmaelba committed:
Update volume3.xml

selmaelba committed:
Update volume3.xml

Tagged paintings from 11/26 and updated header

selmaelba committed:
Merge pull request #33 from selmaelba/volume3

Volume3

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

BenLewis42 committed:
Merge branch 'volume3' of https://github.com/eba-diary/tagged-volumes into volume3

iwiebe7 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

BenLewis42 committed:
Revert "Update volume3.xml"

This reverts commit 9a7b35a87fd58f87d06ba61299ff7959649f081b.

BenLewis42 committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

emmafritzberg commented on diary-volumes/volume3.xml:
```diff
@@ -39,307 +39,308 @@
           <date when="1894-12-26">Wednesday, Dec. 26</date>
         </dateline>
-        <p>We came on board the <orgName>Herodotus</orgName> on the 23d Sunday morning &#8211; a strong south wind had been blowing for days &#8211; but everything had been ready for us for some time. I was tired of the hotel and we hope the south wind would soon be discouraged. But on Monday it holding as strong as ever, we pulled about half a mile away along the bank &#8211; and the next day Xm ...
+        <p>We came on board the <name type="boat" ref="#Herodotus_Boat">Herodotus</name> on the <date when="1894-12-23">23d Sunday morning</date> &#8211; a strong south wind had been blowing for days &#8211; but everything had been ready for us for some time. I was tired of the hotel and we hope the south wind would soon be discouraged. But on <date when="1894-12-24">Monday</date> it holding as s ...
```
Should be `ref=#Herodotus"`

emmafritzberg commented on diary-volumes/volume3.xml:
```diff
@@ -7,7 +7,7 @@
           <persName ref="#Andrews_Emma_B">Emma B. Andrews</persName>
```
@emmafritzberg scrap vol 16 changes in this MR, but keep vol 3 (needs a once-over for correctness)

iwiebe7 committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

emmafritzberg committed:
Merge branch 'main' into volume3

emmafritzberg committed:
Delete Volume16.xml

isabelleschlegel committed:
Update volume3.xml

starting proofread - small changes - changed from Harold to Daniel Jones in my section, orgnames

isabelleschlegel committed:
Merge branch 'volume3' of https://github.com/eba-diary/tagged-volumes into volume3

iwiebe7 committed:
Update volume3.xml

iwiebe7 committed:
Merge branch 'volume3' of https://github.com/eba-diary/tagged-volumes into volume3

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Merge branch 'volume3' of https://github.com/eba-diary/tagged-volumes into volume3

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

small fix removing/updating questions and notes

BenLewis42 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

isabelleschlegel committed:
Update volume3.xml

resolved questions/untagged

iwiebe7 committed:
Update volume3.xml

iwiebe7 committed:
Merge branch 'volume3' of https://github.com/eba-diary/tagged-volumes into volume3

iwiebe7 committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

iwiebe7 committed:
space

iwiebe7 committed:
Update volume3.xml

iwiebe7 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

BenLewis42 committed:
Update volume3.xml

emmafritzberg commented:
Volumes 3 and 15 already have been added
